packages/components/src/components/chat/Chat.vue

```diff
@@ -151,6 +152,9 @@ export default {
     VButton,
   },
   props: {
+    openNewChat: {
+      type: Function,
+    },
```

Comment:
It looks like events were removed, but the pattern is still possible in Vue 3 with an external dependency
https://v3-migration.vuejs.org/breaking-changes/events-api#event-bus

The justification for dropping events seems warranted, but we don't really subscribe to the global (root) event bus, we only use it to publish events to an external source like VSCode. In many cases the events are so deeply nested that passing a function would be cumbersome. Inject/provide is an option, but TypeScript support is pretty poor, at least in Vue 2.